FrontDesk/src/FrontDesk.Blazor/wwwroot/appsettings.json

```diff
@@ -1,7 +1,7 @@
 {
   "baseUrls": {
-    "apiBase": "https://localhost:5001/api/",
-    "webBase": "https://localhost:44394/"
+    "apiBase": "https://localhost:5101/api/",
```

Comment:
If we aren't already, we probably should include a small table in the README listing which apps need to be running and which default ports they're using.